integration/generic-tests/kexec_test.go

```diff
@@ -21,7 +21,6 @@ import (
 // This is possible because the generic initramfs ensures that we mount the
 // testdata directory containing the initramfs and kernel used in the VM.
 func TestMountKexec(t *testing.T) {
-	// TODO: support arm
```

Comment:
"arm" is the GOARCH for 32bit arm, I think you're deleting this because arm64 is here?

Reply:
you are right -- where are are compiling 32bit for nowadays ?